Automattic/wp-calypso
# Pricing page rework: Fix WPCOM styles

#### Proposed Changes

* Fix styles for WPCOM Plans page
* Restores WPCOM theme instead of Jetpack Cloud theme

#### Testing Instructions

- Create a JN site and connect Jetpack with Jetpack Free
- Do any one of these
    * Click on Calypso live link below and wait for the redirect to complete, then goto `/plans/:site?flags=jetpack/pricing-page-rework-v1`
    * or boot up this PR 
        * Run `git fetch && git checkout fix/jp-pricing-rework/wpcom-and-btn-styles`
        * Run `yarn start`
        * Goto `http://calypso.localhost:3000/plans/:site?flags=jetpack/pricing-page-rework-v1`
- Confirm that you see the new pricing UI
- Confirm that the design matches Figma designs
- Now purchase a product, e.g. Backup
- Reload the above plans page on Calypso Blue
- Confirm that the new buttons work as expected.

#### Pre-merge Checklist






Completes: 1202796695664022-as-1202922151448197/f



|BEFORE| AFTER|
|-|-|
|<img width="363" alt="Screenshot 2022-09-05 at 5 46 35 PM" src="https://user-images.githubusercontent.com/18226415/188450127-0e0b061e-d39c-4fec-aba0-3fb14b71fa85.png">|<img width="365" alt="Screenshot 2022-09-05 at 5 47 01 PM" src="https://user-images.githubusercontent.com/18226415/188450136-777731e7-9368-4805-aae8-ffcb5afe5f11.png">|
|<img width="859" alt="Screenshot 2022-09-05 at 5 54 29 PM" src="https://user-images.githubusercontent.com/18226415/188450215-a3858053-8c14-4fa5-a5d3-e87aba5e57c3.png">|<img width="858" alt="Screenshot 2022-09-05 at 5 55 55 PM" src="https://user-images.githubusercontent.com/18226415/188450224-10d8834a-05a2-4ae3-89f9-3676d6aa9fe9.png">|
|<img width="319" alt="Screenshot 2022-09-05 at 5 56 37 PM" src="https://user-images.githubusercontent.com/18226415/188450296-95df9745-151b-403d-b0b6-6ad08f5b848e.png">|<img width="320" alt="Screenshot 2022-09-05 at 5 57 36 PM" src="https://user-images.githubusercontent.com/18226415/188450302-a2ba79b2-f928-4064-a250-7c8a658b7f13.png">|

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202922151448197
